src/test/scala/com/github/timgent/dataflare/metrics/MetricCalculatorTest.scala

```diff
@@ -181,4 +181,56 @@ class MetricCalculatorTest extends AnyWordSpec with DatasetSuiteBase with Matche
       )
     }
   }
+
+  "MinValueMetricCalculator" should {
+    lazy val ds = Seq(
+      NumberString(1, "a"),
+      NumberString(2, "b"),
+      NumberString(3, "c"),
+      NumberString(3, "d"),
+      NumberString(3, "d") // not distinct
```

Comment:
Don't need this line - distinctness isn't important for min value metric

Reply:
Have removed this line